compiler/rustc_monomorphize/src/collector.rs

```diff
@@ -1296,12 +1296,12 @@ impl<'v> RootCollector<'_, 'v> {
             return;
         };
 
-        let start_def_id = match self.tcx.lang_items().require(LangItem::Start) {
-            Ok(s) => s,
-            Err(lang_item_err) => {
+        let start_def_id = match self.tcx.lang_items().start_fn() {
+            Some(s) => s,
+            None => {
                 self.tcx
                     .sess
-                    .emit_fatal(RequiresLangItem { lang_item: lang_item_err.0.name().to_string() });
+                    .emit_fatal(RequiresLangItem { lang_item: LangItem::Start.name().to_string() });
             }
         };
```

Comment:
This should probably use https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/context/struct.TyCtxt.html#method.require_lang_item

Reply:
done